spec/unit/matrixrtc/mocks.ts

```diff
@@ -49,9 +53,10 @@ export function makeMockRoomState(
 export function mockRTCEvent(
     memberships: CallMembershipData[],
     roomId: string,
-    getLocalAge: (() => number) | undefined,
+    localAge: number | null,
+    localTimestamp: number | null = null,
```

Comment:
could you add a doc-comment here and to `makeMockRoomState` documenting how these two interact? (My understanding is that `localAge` is ignored if `localTimestamp` is non-null? But if you could make that explicit that would be great).

Reply:
looks like this is no longer relevant because the functions in question now only have a `localAge` (rather than a `localTimestamp`. I wouldn't have minded seeing a doc-comment explaining what `localAge` does anyway, but it's not too important